src/utils/display-observer.js

```diff
@@ -248,6 +250,20 @@ function computeDisplay(observations, isDocVisible) {
   return observations.reduce(displayReducer);
 }
 
+/**
+ * @param {!VisibilityState} visibilityState
+ * @return {boolean}
+ */
+function computeDocDisplay(visibilityState) {
+  return (
+    visibilityState == VisibilityState.VISIBLE ||
+    // The document is still considered "displayed" or at least "displayable"
+    // when it's hidden (tab is switched). Only prerender/paused/inactive
+    // states require pause of resources.
+    visibilityState == VisibilityState.HIDDEN
```

Comment:
This state is also used when a viewer swipes to the next doc. Shouldn't that be marked as not displayed?

Reply:
Not this one. I checked and in the viewer, the sequence of states is: "visible" -> "paused" -> "inactive".

Reply:
Yup, tested and this works correctly. Viewer swipe leads to `INACTIVE`, and tab change leads to `HIDDEN`.

Should `PAUSED` be included here? I think yes. And what about `PRERENDER`? I think that could be likely no, because someone might depend on it and accidentally leak privacy.

Reply:
I believe, yes, because PAUSE is explicitly made for this use-case. I think the way this happens is that PAUSED is issued immediately and INACTIVE some time later to avoid too many unlayouts while the new page is loaded.